cg/clients/freshdesk/models.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class Ticket(BaseModel):
+    """Freshdesk ticket."""
+
+    association_type: int | None = None
+    attachments: list[dict[str, str]] = []
+    cc_emails: list[str] = []
+    company_id: int | None = None
+    custom_fields: dict | None = None
+    deleted: bool | None = None
+    description: str | None = None
+    description_text: str | None = None
+    due_by: datetime | None = None
+    email: str | None = None
+    email_config_id: int | None = None
+    facebook_id: str | None = None
+    fr_due_by: datetime | None = None
+    fr_escalated: bool | None = None
+    fwd_emails: list[str] = []
+    group_id: int | None = None
+    id: int | None = None
+    is_escalated: bool | None = None
+    name: str | None = None
+    phone: str | None = None
+    priority: int = 1
+    product_id: int | None = None
+    reply_cc_emails: list[str] = []
+    requester_id: int | None = None
+    responder_id: int | None = None
+    source: int = 2
+    source_additional_info: str | None = None
+    spam: bool | None = None
+    status: int = 3
```

Comment:
Is there a reason for the default values 1,2,3 here? Seems very cryptic to me.

Reply:
Yes, I should probably make them constants. Source: 2 is from the API (weird that one can choose that 😅 ). Status 3 is open, and priority 1 is medium i think. But that should be clarified

Reply:
Added constants in [3dca545](https://github.com/Clinical-Genomics/cg/pull/3317/commits/3dca545ffb56567f9685a269e81d154581de9f88)